tasks/extract-binaries-from-image/extract-binaries-from-image.yaml

```diff
@@ -83,7 +83,13 @@ spec:
           for DIGEST in $(jq -r ".layers[].digest" manifest.json)
           do
               FILE=${DIGEST#sha256:}
-              tar -xzvf "$FILE"
+              # Check if the archive contains the $IMAGE_PATH dir
+              if tar -tf "$FILE" | grep -q '^$IMAGE_PATH/'; then
+                echo "Extracting $FILE..."
+                tar -xzvf "$FILE"
+              else
+                echo "$FILE doesn't contain the $IMAGE_PATH dir"
```

Comment:
Can you add ", skipping" to the echo? Otherwise it may not be clear what that message means in the log.

Reply:
Added